### ISSUE TITLE
request: Use cache for .tgs (Animated sticker files)

### ISSUE THREAD

**Issue by sentinm** [tjtanjin/simple-media-converter#4]
> From @SimpleMediaConverterBot
> This bot currently does not support animated stickers :(

### Use cache for .tgs files (animated telegram stckers), because it take alot time to render a .gif for a .tgs file with high encoding and without background. Some examples are given below :point_down: 

* From @Stickerdownloadbot (Which don't use cache and no encoding) :point_down: 
<img src="https://user-images.githubusercontent.com/79321771/118355928-6affd000-b590-11eb-8e77-d4110d547abb.gif" width="200">

* From @tgstogifbot (Which uses cache and have high encoding) :point_down: 
<img src="https://user-images.githubusercontent.com/79321771/118264526-717d4180-b4d5-11eb-9b16-6c5842efdd48.gif" width="200">

 ### If you are willing to add this feature better use the library used in @tgstogifbot
* Library :- [Tgs to Gif](https://github.com/ed-asriyan/tgs-to-gif)

**Comment by tjtanjin** [comment]
I know the issue has been closed but I thought I'd drop a comment here. Also apologies I actually just saw this 😅

I was updating the project briefly to bring it up to date and decided to throw in support for animated stickers as well. I've noticed that the stickers currently shows a black background when converted instead of keeping the transparency. The library you shared doesn't seem to have this issue (though it requires additional setup outside of this repository).

In the near future (hopefully), I'll look to improve the conversion. Alternatively, I'm always open to someone else coming along and adding new features x)
